cli/tsc/99_main_compiler.js

```diff
@@ -8,11 +8,22 @@
 // that is created when Deno needs to type check TypeScript, and in some
 // instances convert TypeScript to JavaScript.
 
-// Removes the `__proto__` for security reasons.
+// Disables setting `__proto__` and emits a warning instead, for security reasons.
```

Comment:
I believe it should stay disabled in the TSC implementation. There won't be any user code executed here that might require it.

Reply:
`chalk@3` won't work with this change (chalk uses the setter) -- but it _should_ emit a warning which will help anyone using a package dependent on chalk know why there's an error similar to the one shown in `pm2`

Reply:
Do you know of any package that only uses getter?

Reply:
I don't happen to know any -- and I'm not sure why any package would want to access a prototype when it cannot be modified --  but I don't see any harm in leaving it in. I could make a test npm package that accesses __proto__ if needed (although I don't see how this is npm specific)